conan/tools/qbs/qbsdeps.py

```diff
@@ -55,10 +55,11 @@ def render(self):
 class _QbsDepGenerator:
     """ Handles a single package, can create multiple modules in case of several components
     """
-    def __init__(self, conanfile, dep, build_bindirs):
+    def __init__(self, conanfile, dep, build_bindirs, use_cmake_file_name):
```

Comment:
Probably no need of the new argument, if we are going to use properties, maybe the approach should be similar to CMakeDeps and allow a local QBSDeps.set_property() that can override the upstream property, and also allows defining the property locally. This would be general for any property that the generator might use in the future too.